lib/graph.ml

```diff
@@ -0,0 +1,90 @@
+type color = Red | Blue | Green | Yellow
+type position = int * int
+type vertex = position * color
+type neighbors = vertex list
+type graph = int * (vertex * neighbors) list
```

Comment:
Maybe give names to your variables. What is int?

Reply:
it's for the time, but it's true that a structure might be better, I'll modify it.